### PR TITLE
Testing improvements

### PR DESCRIPTION
**Pull request by murrant**
Please review.  Don't merge just yet.

### CODE REVIEW

File: tests/db/AddDefaultDataTest.php [hunk 1]
```diff
@@ -18,8 +18,8 @@ public function testAdddbSchemaVersion()
     public function testaddDeviceData()
     {
         $data_array = [
-                ['hostname' => 'localhost', 'sysName' => 'mysystem', 'version' => '1.1', 'hardware' => 'Intel x64', 'location' => 'Some place in the world', 'status' => 1, 'status_reason' => ''],
-                ['hostname' => 'remotehost', 'sysName' => 'someonessystem', 'version' => '2.2', 'hardware' => 'AMD x64', 'location' => 'Some other place in the world', 'status' => 0, 'status_reason' => 'icmp'],
+                ['hostname' => 'localhost', 'ip' => '127.0.0.1', 'sysName' => 'mysystem', 'version' => '1.1', 'hardware' => 'Intel x64', 'location' => 'Some place in the world', 'status' => 1, 'status_reason' => ''],
+                ['hostname' => 'remotehost', 'ip' => '127.0.0.2', 'sysName' => 'someonessystem', 'version' => '2.2', 'hardware' => 'AMD x64', 'location' => 'Some other place in the world', 'status' => 0, 'status_reason' => 'icmp'],
```
Comment (laf) [inline tests/db/AddDefaultDataTest.php]: ip column is a binary one so this needs to use inet-aton conversion

**Comment by laf** [comment]
Thanks @murrant. Does dropping the index name still create the tables correctly?

**Comment by murrant** [comment]
@laf, yes, Laravel generates the index name.  The problem was that many indexes had the same name, which isn't allowed in sqlite.  I'm not sure there would be any reason to know the name of the index.

**Comment by laf** [comment]
Glad you got it working :)

The /about page is failing phpunit tests on this one.

**Comment by murrant** [comment]
@laf Yeah, I can't figure out the about page test.  It fails on my system even without these changes.  Any ideas?

**Pull request merged**